arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -49,14 +47,23 @@ public async Task<DeleteCollectionResponse> DeleteCollectionAsync(string collect
             throw new ApiErrorException(error);
         }
 
-        public async Task PutCollectionTruncateAsync(string collectionName)
+        /// <summary>
+        /// Truncates a collection, i.e. removes all documents in the collection.
+        /// PUT/_api/collection/{collection-name}/truncate
+        /// </summary>
+        /// <param name="collectionName"></param>
+        /// <returns></returns>
+        public async Task<TruncateCollectionResult> TruncateCollectionAsync(string collectionName)
         {
-            var response = await _transport.PutAsync(_collectionApiPath + "/" + collectionName + "/truncate", null);
-            if (response.IsSuccessStatusCode)
+            using (var response = await _transport.PutAsync(_collectionApiPath + "/" + collectionName + "/truncate", null))
             {
-                return;
+                if (response.IsSuccessStatusCode)
+                {
+                    var stream = await response.Content.ReadAsStreamAsync();
+                    return DeserializeJsonFromStream<TruncateCollectionResult>(stream, true, false);
+                }
+                throw await GetApiErrorException(response);
```

Comment:
In the other method of this client we do:

```c#
                var stream = await response.Content.ReadAsStreamAsync();
                if (response.IsSuccessStatusCode)
                {
                    return DeserializeJsonFromStream<PostCollectionResponse>(stream, true, false);
                }
                throw new ApiErrorException(
                    DeserializeJsonFromStream<ApiErrorResponse>(stream, true, false));
```

Should we pick one preferred method? While it does the same thing, I think it's good to use a consistent approach across different methods of one client. Is it just me?

Reply:
Yes, I implemented GetApiErrorException after the other implementation you refer to. We can change to use the same pattern. I don't regard it as a big priority though.